datadog/resource_datadog_synthetics_test_.go

```diff
@@ -316,6 +359,9 @@ func newSyntheticsTestFromLocalState(d *schema.ResourceData) *datadog.Synthetics
 		minLocationFailed, _ := strconv.Atoi(attr.(string))
 		options.SetMinLocationFailed(minLocationFailed)
 	}
+	if attr, ok := d.GetOk("options.accept_self_signed"); ok {
+		options.SetAcceptSelfSigned(attr.(string) == "true")
```

Comment:
Any reason not to just assign this to `attr.(string)` Should we also safely type assert this?

Reply:
Not sure, can't wrap my head around it. It's value is "1" or "0" in TF 0.11, even though is never set to such value